### DISCUSSION TITLE
Error - Incorrect Request

### DISCUSSION THREAD

**Discussion by douglasatmc** [YetiForceCompany/YetiForceCRM#17003] [General]
I suddenly started to have an issue with the import feature within yetiforce.  I navigate to my module that I need to import data into, select the actions button, finally selecting import.
I am than greeted by an error:
"incorrect Action: there was an error, contact your administrator"
The problem is, I do not see any error entries in any logs, this is occuring will all modules, and I have just finalized my module to actually be able to import data, and am now ffacing this issue. Please note, I am no CRM pro, i am no database genius, I am very new to this, and build with the module editors, as I do not use code at all. PLEASE HELP, my not for profit needs to get this data uploaded!

**Comment by ghost** [comment]
Have you checked well if the data you are carrying contains errors?

**Comment by rskrzypczak** [comment]
Enable logs and check: https://doc.yetiforce.com/developer-guides/debug/
and also check this: https://github.com/YetiForceCompany/YetiForceCRM/discussions/16777
